composablejs/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "composable",
+  "version": "0.1.0",
+  "private": true,
+  "author": "@Dom-Roth",
+  "license": "Public Domain",
```

Comment:
Not sure if this is a good idea

Reply:
The author is generally intended to mention a single person (e.g. creator/maintainer), instead of a group or company.
But it's not a required parameter, so I'll just remove it. :+1: